client/src/components/geneExpression/geneSet.js

```diff
@@ -81,25 +81,33 @@ class GeneSet extends React.Component {
   renderGenes() {
     const { setName, setGenes, setGenesWithDescriptions } = this.props;
 
-    return setGenes.map((gene) => {
-      const { geneDescription } = setGenesWithDescriptions.get(gene);
-
-      return (
-        <Gene
-          key={gene}
-          gene={gene}
-          geneDescription={geneDescription}
-          geneset={setName}
-        />
-      );
-    });
+    return (
+      <div data-testclass="gene-set-genes">
+        {setGenes.map((gene) => {
+          const { geneDescription } = setGenesWithDescriptions.get(gene);
+
+          return (
+            <Gene
+              key={gene}
+              gene={gene}
+              geneDescription={geneDescription}
+              geneset={setName}
+            />
+          );
+        })}
+      </div>
+    );
   }
 
   render() {
     const { setName, setGenes, genesetDescription } = this.props;
     const { isOpen } = this.state;
     const genesetNameLengthVisible = 150; /* this magic number determines how much of a long geneset name we see */
     const genesetIsEmpty = setGenes.length === 0;
+    let testClass = "geneset-expand";
+
+    if (setName.includes("Pop1 high")) testClass = "pop-1-geneset-expand";
```

Comment:
this is very, very fragile.  If you change the name in the reducer, it will break the tests.  Also, the reducer really should not contain the UI (ie, the population names) - they should be in the component.

I think this information should be passed as flags, not inferred from UI/presentation strings.